$NetBSD: patch-ab,v 1.3 2009/01/05 12:15:17 he Exp $

Changes:
 o More general adaptation for BSDs
 o Install scripts in DESTINSTALLSITESCRIPT instead of DESTINSTALLSCRIPT

--- lib/ExtUtils/MM_Unix.pm.orig	2008-02-29 01:06:28.000000000 +0100
+++ lib/ExtUtils/MM_Unix.pm
@@ -931,7 +931,7 @@ $(INST_DYNAMIC): $(OBJECT) $(MYEXTLIB) $
 
     my $libs = '$(LDLOADLIBS)';
 
-    if (($Is{NetBSD} || $Is{Interix}) && $Config{'useshrplib'} eq 'true') {
+    if ($Is{BSD} && $Config{'useshrplib'} eq 'true') {
 	# Use nothing on static perl platforms, and to the flags needed
 	# to link against the shared libperl library on shared perl
 	# platforms.  We peek at lddlflags to see if we need -Wl,-R
@@ -2119,7 +2119,7 @@ pure_perl_install ::
 		$(INST_LIB) $(DESTINSTALLPRIVLIB) \
 		$(INST_ARCHLIB) $(DESTINSTALLARCHLIB) \
 		$(INST_BIN) $(DESTINSTALLBIN) \
-		$(INST_SCRIPT) $(DESTINSTALLSCRIPT) \
+		$(INST_SCRIPT) $(DESTINSTALLSITESCRIPT) \
 		$(INST_MAN1DIR) $(DESTINSTALLMAN1DIR) \
 		$(INST_MAN3DIR) $(DESTINSTALLMAN3DIR)
 	$(NOECHO) $(WARN_IF_OLD_PACKLIST) \
